packages/sommelier.rb

```diff
@@ -2,106 +2,256 @@
+    Dir.chdir ("vm_tools/sommelier") do
+      system "DESTDIR=#{CREW_DEST_DIR} ninja -C build install"
+      Dir.chdir ("build") do
+        FileUtils.mv "#{CREW_DEST_PREFIX}/bin/sommelier", "#{CREW_DEST_PREFIX}/bin/sommelier.elf"
+        system "install -Dm755 sommelier_sh #{CREW_DEST_PREFIX}/bin/sommelier"
+        system "install -Dm755 sommelierd #{CREW_DEST_PREFIX}/sbin/sommelierd"
+        system "install -Dm755 startsommelier #{CREW_DEST_PREFIX}/bin/startsommelier"
+        FileUtils.ln_sf 'startsommelier', "#{CREW_DEST_PREFIX}/bin/initsommelier"
+        system "install -Dm755 stopsommelier #{CREW_DEST_PREFIX}/bin/stopsommelier"
+        system "install -Dm755 restartsommelier #{CREW_DEST_PREFIX}/bin/restartsommelier"
+        system "install -Dm755 sommelierrc #{CREW_DEST_PREFIX}/etc/sommelierrc"
+        system "install -Dm644 .sommelier.env #{CREW_DEST_HOME}/.sommelier.env"
+
+      end
     end
   end
-
+  
   def self.postinstall
     puts
     puts "To complete the installation, execute the following:".lightblue
-    puts "echo '# Sommelier environment variables + daemon' >> ~/.bashrc".lightblue
+    puts "echo '# Sommelier envronment variables + daemon' >> ~/.bashrc".lightblue
     puts "echo '# See https://github.com/dnschneid/crouton/wiki/Sommelier-(A-more-native-alternative-to-xiwi)' >> ~/.bashrc".lightblue
-    puts "echo 'if [ ! -d /tmp/.X11-unix ]; then' >> ~/.bashrc".lightblue
-    puts "echo 'mkdir /tmp/.X11-unix' >> ~/.bashrc".lightblue
-    puts "echo 'fi' >> ~/.bashrc".lightblue
-    puts "echo 'sudo chmod -R 1777 /tmp/.X11-unix' >> ~/.bashrc".lightblue
-    puts "echo 'sudo chown root:root /tmp/.X11-unix' >> ~/.bashrc".lightblue
+    puts "echo 'set -a && source ~/.sommelier.env && set +a' >> ~/.bashrc".lightblue
     puts "echo 'startsommelier' >> ~/.bashrc".lightblue
     puts "source ~/.bashrc".lightblue
     puts
-    puts "To start the sommelier daemon, run 'startsommelier'".lightblue
-    puts "To stop the sommelier daemon, run 'stopsommelier'".lightblue
-    puts
     puts "To adjust environment variables, edit ~/.sommelier.env".lightblue
+    puts "e.g. You may need to adjust the SCALE environment variable to get the correct screen size.".lightblue
     puts
-    puts "You may need to adjust the SCALE environment variable to get the correct screen size.".lightblue
+    puts "To start the sommelier daemon, run 'startsommelier'".lightblue
+    puts "To stop the sommelier daemon, run 'stopsommelier'".lightblue
+    puts "To restart the sommelier daemon, run 'restartsommelier'".lightblue
     puts
     puts "Please be aware that gui applications may not work without the sommelier daemon running.".orange
     puts
-    puts "If you are upgrading from an earlier version of sommelier, run".orange
-    puts "'stopsommelier && startsommelier' or logout and login again.".orange
+    puts "The sommelier daemon may also have to be restarted with 'restartsommelier' after waking your device.".red
```

Comment:
Probably should use orange instead of red here.  Red is usually reserved to indicate an error.

Reply:
done.